app/sdk/emailNotifications/EmailUtils.java

```diff
@@ -0,0 +1,47 @@
+package sdk.emailNotifications;
+
+import play.libs.ws.WSRequest;
+import sdk.AppTreeSource;
+import sdk.utils.Constants;
+import sdk.utils.JsonUtils;
+
+/**
+ * Created by alexis on 12/5/16.
+ */
+public class EmailUtils implements AppTreeSource {
+    private String coreURL;
+    private static String defaultCoreURL = "https://services1.apptreesoftware.com";
```

Comment:
@alexisandreason The `defaultCoreUrl` seems like something we should put in the application.conf